fanaSystem/fanaAuthenticator/views.py

```diff
@@ -119,4 +157,4 @@ def handle_customer_order(request):
         except requests.RequestException as e:
             return JsonResponse({'status': 'error', 'message': str(e)}, status=500)
     
-    return JsonResponse({'status': 'error', 'message': 'Invalid request method'}, status=400)
+    return JsonResponse({'status': 'error', 'message': 'Invalid request method'}, status=400)
```

Comment:
Ditto, ensure that the file ends with an empty line

Reply:
Done.